R/prophet.R

```diff
@@ -478,7 +478,7 @@ Prophet <- function(jaspResults, dataset = NULL, options) {
 
     criLevel <- (1-options[["summaryCredibleIntervalWidth"]])/2
 
-    overtitle <- gettext(paste0(options[["summaryCredibleIntervalWidth"]]*100, "% CI"))
+    overtitle <- paste0(options[["summaryCredibleIntervalWidth"]]*100, "% CI")
```

Comment:
`gettextf("%i%% CI", options[["summaryCredibleIntervalWidth"]]*100)`

Reply:
Basically if you want to concatenate strings and also have them translated (which we do in this case) you need to use `gettextf()`.

Reply:
True, I forgot that `% CI` will be translated!

Reply:
I see you did the `ciTitle`, don't forget to do the `overtitle` too :)
Afterwards this can be merged.

Reply:
Sorry, I was a bit distracted yesterday :D

Reply:
No worries